package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ciscospark/react-widgets",
-  "version": "0.2.37",
+  "version": "0.2.38",
```

Comment:
This change indicates that the master branch is quite out of date. Could you rebase this PR on the latest master branch?

Reply:
@adamweeks Updated to latest master branch